src/main/java/com/webonise/rbs/controller/UserController.java

```diff
@@ -0,0 +1,77 @@
+package com.webonise.rbs.controller;
+
+import com.sun.org.apache.regexp.internal.RE;
+import com.webonise.rbs.entity.User;
+import com.webonise.rbs.repository.UserRepository;
+import com.webonise.rbs.service.UserService;
+import com.webonise.rbs.service.UserServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import java.util.ArrayList;
+
+@Controller
+@RequestMapping("/user")
+public class UserController {
+
+    @Autowired
+    UserServiceImpl userService;
+
+    @GetMapping(value = "/",produces = MediaType.TEXT_HTML_VALUE)
+    public String savePage(Model model) {
+        model.addAttribute("user", new User());
+        model.addAttribute("allUsers", userService.getAllUsers());
+        return "users";
+    }
+
+    @PostMapping("/save")
```

Comment:
Mapping type Post signifies that you are saving something, so need to have uri /save.
https://hackernoon.com/restful-api-designing-guidelines-the-best-practices-60e1d954e7c9